src/usePopper.ts

```diff
@@ -21,53 +23,41 @@ export type Placement = Popper.Placement;
 export type VirtualElement = Popper.VirtualElement;
 export type State = Popper.State;
 
+export type OffsetsFunction = (details: {
+  popper: Popper.Rect;
+  reference: Popper.Rect;
+  placement: Placement;
+}) => [number | null | undefined, number | null | undefined];
+
+export type Offset =
```

Comment:
**nitpick:** `OffsetsFunction` v `Offset` is odd, no? it should be consistently singular or plural.

Reply:
oops yeah, i just copied it from popper